rails/config/routes.rb

```diff
@@ -332,6 +330,8 @@
         get :set_private_before_matedit
         get :copy
         get :edit_basic
+        match 'collections' => 'external_activities#edit_collections', :via => :get
+        match 'collections' => 'external_activities#update_collections', :via => :post
```

Comment:
I looked at this more. And this should be `:put` instead of `:post`.  This might require a change in the view so it uses a put method instead of a post method.